entries/c/cordial.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Cordial": {
+    "domain": "cordial.com",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support.cordial.com/hc/en-us/articles/360004846512",
+    "recovery": "https://support.cordial.com/hc/en-us/requests/new",
```

Comment:
This page isn't really helpful by itself - the link is provided on the doc page as well, so that should be enough.
```suggestion
```